documentation/cobigen-core_configuration.asciidoc

```diff
@@ -378,6 +378,113 @@ However, if we want to reference an increment that it is not defined inside our
 
 The ref string is split using as delimiter `::`. The first part of the string, is the `trigger_id` to reference. That trigger contains an `increment_id`. Currently, this functionality only works when both templates use the same kind of input file.
 
+== Configuration Upgrade
+
+`CobiGen 2021.12.007` introduces an updated configuration folder structure:
+
+```
+template-sets
+ |- downloaded
+   |- template-set1.jar
+   |- template-set2.jar
+   |- template-set-utilities.jar
+ |- adapted
+   |- template-set1
+     |- src
+       |- main
+         |- templates
+           |- templates
+             |- context.xml
+             |- templates.xml
+   |- template-set2
+```
+
+The updated structure splits the old structure, which consisted of multiple pom and context files,
+into multiple files corresponding to every template set that will be created.
```

Comment:
I think you should rewrite this part
The old structure did not have multiple pom and context files, but 1 oversized version each.
The second part should maybe be something like "splits [...] into multiple directories - one for each template set - each with their own atomized/small/more understandable version so you can download them individually instead of all at once"

In short: try to write the core things about what changed and why this matters for the user

Reply:
Thank you I have updated it